### ISSUE TITLE
Wrong permissions for kube-*.json

### ISSUE THREAD

**Issue by antoineco** [neuvector/kubernetes-cis-benchmark#5]
The following (correct) permissions yield a warning:
```
-rw-r-----. 1 root root 1796 Jun 30 19:04 kube-apiserver.json
-rw-r-----. 1 root root 1873 Jun 30 18:58 kube-controller-manager.json
-rw-r-----. 1 root root 1406 Jun 30 18:39 kube-proxy.json
-rw-r-----. 1 root root 1440 Jun 30 18:39 kube-scheduler.json
```

**Comment by yanguo2111** [comment]
fixed, thanks.

**Comment by garyduan** [comment]
Fixed by #8
